tests/functional/spec/dob_date.spec.js

```diff
@@ -0,0 +1,22 @@
+import DateOfBirthPage from "../generated_pages/dob_date/date-of-birth.page";
+import UnderSixteenPage from "../generated_pages/dob_date/under-sixteen.page";
+
+describe("Date of birth check", () => {
+  beforeEach("Load the survey", () => {
+    browser.openQuestionnaire("test_dob_date.json");
+  });
+  it("Given Iam completing a date question, when I value less than 16 years, Iam routed to under 16 page", () => {
```

Comment:
Just needs a tweak.

"Given I am completing a date question, when I enter a value less than 16 years, Then I am routed to the under 16 page"